index.html

```diff
@@ -210,11 +228,10 @@
             scoreOutput.textContent = scoreSlider.value;
             scoringSection.appendChild(scoreOutput);
 
-            app.appendChild(scoringSection);
-
             scoreSlider.addEventListener("input", () => {
                 scoreOutput.textContent = scoreSlider.value;
             });
+            app.appendChild(scoringSection);
```

Comment:
Finally, add the cards and scoringSection to the app container. Add the following code before the app.appendChild(addButton); line in the <script> section: